src/components/FlightTimeSelector/FlightTimeSelector.tsx

```diff
@@ -0,0 +1,163 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import styles from "./FlightTimeSelector.module.css";
+import "react-calendar/dist/Calendar.css";
+import Icon from "../Icon";
+import DatePicker from "react-calendar";
+import { useState } from "react";
+
+const FlightTimeSelector = () => {
+  type Value = Date | null | [Date, Date];
```

Comment:
Place in definitions file and give a more specific name (what kind of `Value` is this?)